src/test/java/com/faforever/client/test/AbstractPlainJavaFxTest.java

```diff
@@ -32,6 +32,9 @@
 
   public AbstractPlainJavaFxTest() {
     root = new Pane();
+    Thread.currentThread().setUncaughtExceptionHandler((thread, e) -> {
+      throw (RuntimeException) e;
```

Comment:
Should we not log them???

Reply:
This code makes it so that tests fail like as if you throw any other exception inside a test case (which will automatically cause the test framework to log the exception). So logging here would produce double stacktrace.

Reply:
I read the issue now and cool I really hated that behaviour